packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html

```diff
@@ -598,6 +613,15 @@ <h2>
                                     data-translate-values="{ price: $ctrl.getFlavorPrice(flavor) }"
                                 >
                                 </strong>
+                                <ovh-manager-catalog-price
+                                    data-price="$ctrl.flavorPrice"
+                                    data-tax="$ctrl.flavorPriceTax"
+                                    data-user="$ctrl.user"
+                                    data-perform-rounding="false"
+                                    data-maximum-fraction-digits="3"
+                                >
+                                </ovh-manager-catalog-price>
+                                <strong>/heure</strong>
```

Comment:
**question:** Shouldn't this content be translated?

Reply:
Definitely, yes ! As we use `ovh-manager-catalog-price`, we can set the `interval` attribute to `hour` to do so. I've changed it here: https://github.com/ovh/manager/pull/6963/commits/e436aa861fc04db0f1e66f0a26433c7b65a3895b
Thanks !